workspaces/backend/cmd/main.go

```diff
@@ -34,7 +34,10 @@ func main() {
 
 	logger := slog.New(slog.NewTextHandler(os.Stdout, nil))
 
-	app := application.NewApp(cfg, logger)
+	app, err := application.NewApp(cfg, logger)
+	if err != nil {
+		logger.Error(err.Error())
```

Comment:
This should probably exit with non-zero code rather than continuing.

```suggestion
		logger.Error(err.Error())
		os.Exit(1)
```

Reply:
Yeah that was a mistake.